src/components/switch/switch.spec.tsx

```diff
@@ -581,6 +593,34 @@ describe("Switch", () => {
     });
   });
 
+  describe("New validations", () => {
+    const validationTypes = ["error", "warning"] as const;
```

Comment:
**comment** (non-blocking): is there a good reason not to also use `info` in these tests?

Reply:
There is no `info` in the new validations. I'll send you a link to the Figma designs in a private chat.